pkg/ottl/contexts/internal/ids/ids.go

```diff
@@ -0,0 +1,46 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ids // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/internal/ids"
+
+import (
+	"encoding/hex"
+	"errors"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+)
+
+func ParseSpanID(spanIDStr string) (pcommon.SpanID, error) {
```

Comment:
Do these need to be moved to their own pacakge?  Could they live in `ottlcommon` in an `ids.go`?

Reply:
It can. Moved.